PKGBUILD

```diff
@@ -0,0 +1,47 @@
+# Maintainer: Cosmin Gabriel Georgescu <cosmingg2013@gmail.com>
+pkgname="duckscript"
+pkgver=0.8.20
+pkgrel=1
+pkgdesc="Simple, extendable and embeddable scripting language."
+
+# https://github.com/sagiegurari/duckscript#installation-binary-release
+arch=("x86_64")
+
+url="https://sagiegurari.github.io/duckscript/"
+
+# https://github.com/sagiegurari/duckscript/blob/master/LICENSE
+license=("Apache")
+
+makedepends=("cargo")
+
+source=("$pkgname-$pkgver.tar.gz::https://github.com/sagiegurari/duckscript/archive/$pkgver.tar.gz")
+
+sha512sums=("7ffe4ad2d1bb54753d7c262981bf31394404dd71a75919188a5f658de39b9b9215d057da382e220433d866369417e74feaef3ab4ac80df5214b046284f5f840d")
+
+b2sums=("869859f0d7e21abf1f3e1041e6f7056c25d4e6341bab19788c95c609da60cff5add66ec113aa1f2ff9557596537728a6255ec65c280668d9d3e8e785d042f616")
+
+prepare(){
+	export RUSTUP_TOOLCHAIN=stable
+	cargo fetch --locked --target "$arch-unknown-linux-gnu"
```

Comment:
It crossed my mind just now, should the libc be `musl` or `gnu` ? For each release, I've noticed you only publish the `x86_64-unknown-linux-musl` version and never a `gnu` one.

Reply:
i don't mind publishing all of them.

Reply:
Ah, only if you want to; it doesn't help the script in any way. Thought there might be a performance reason or something you found.